### PR TITLE
Update the description of Message liquid drop

### PR DESCRIPTION
**Pull request by mayorova**
Follow-up for https://github.com/3scale/porta/pull/3673#discussion_r1460350656

### CODE REVIEW

File: lib/developer_portal/lib/liquid/drops/message.rb [hunk 1]
```diff
@@ -5,7 +5,7 @@ class Message < Drops::Model
 
       allowed_name :message, :messages, :reply
 
-      desc "Returns the ID of the model (depending on controller that is message or message recipient)."
+      desc "Returns the ID of the received or sent message."
```
Comment (akostadinov) [inline lib/developer_portal/lib/liquid/drops/message.rb:8]: Can we then add a normal code comment here in this case, to make the above known?

**Review by akostadinov — COMMENTED** [review]
I wanted to merge the previous one to get test suite more reliable. I thought you would wan to change this and that makes sense.

Can we also add the message vs recipient id test here? Like update recipient  id to something different than message id?

File: lib/developer_portal/lib/liquid/drops/message.rb (hunk 1 above)
Comment (mayorova) [reply lib/developer_portal/lib/liquid/drops/message.rb:8]: There is already a comment here: https://github.com/3scale/porta/blob/master/lib/developer_portal/lib/liquid/drops/message.rb#L3

It is really close to that `#id` method :)

**Review by jlledom — APPROVED** [review]

**Comment by mayorova** [comment]
> Can we also add the message vs recipient id test here? Like update recipient id to something different than message id?

Not sure what you mean exactly @akostadinov

**Pull request merged**